Dockerfiles/Cpp/Dockerfile

```diff
@@ -84,6 +84,12 @@ RUN apt-get --fix-missing update && apt-get upgrade -y && \
         update-alternatives --install /usr/bin/clang-format clang-format /usr/bin/clang-format-14 100 && \
         update-alternatives --install /usr/bin/clang-tidy clang-tidy /usr/bin/clang-tidy-14 100
 
+# Install CLI
+ARG TARGETARCH
+COPY ./bin/velocitas-cli-$TARGETARCH /usr/bin/velocitas
+RUN chmod +x /usr/bin/velocitas
```

Comment:
can't we do this before commiting? or is it not persistent?

Reply:
Changed file permissions and removed this